src/mainwindow.cpp

```diff
@@ -164,6 +164,20 @@ MainWindow::MainWindow(QWidget* parent)
 
     connect(m_resultsPage, &ResultsPage::navigateToCode, this, &MainWindow::navigateToCode);
     ui->fileMenu->addAction(KStandardAction::open(this, SLOT(onOpenFileButtonClicked()), this));
+
+    auto openNewWindow = new QAction(QIcon::fromTheme(QStringLiteral("document-open")), tr("Open in new window"), this);
+    openNewWindow->setShortcut(Qt::Key_O | Qt::ControlModifier | Qt::ShiftModifier);
+    connect(openNewWindow, &QAction::triggered, this, [this] {
+        const auto fileName = QFileDialog::getOpenFileName(this, tr("Open File"), QDir::currentPath(),
+                                                           tr("Data Files (perf*.data perf.data.*);;All Files (*)"));
+        QProcess process;
+        process.setProgram(qApp->applicationFilePath());
+        process.setArguments({fileName});
```

Comment:
what about other args, i.e. what happens when we do:

```
hotspot --sysroot foo perf.data
```

and then try to open another window, we would lose the sysroot then, no?

Reply:
in theory it should read it from the settings file

Reply:
now it is read from settings